packages/react/src/components/Popup/PopupContent.tsx

```diff
@@ -51,10 +52,7 @@ export interface PopupContentProps
   pointing?: boolean
 
   /** A ref to a pointer element. */
-  pointerRef?: PopperChildrenProps['arrowProps']['ref']
-
-  /** An object with positioning styles fof a pointer. */
-  pointerStyle?: PopperChildrenProps['arrowProps']['style']
+  pointerRef?: React.RefObject<Element>
```

Comment:
```suggestion
  pointerRef?: React.Ref<Element>
```

As we accept functions there, too

Reply:
done, thanks